project/components/resource_request/models.py

```diff
@@ -48,5 +50,7 @@ class ResourceRequest(DBModel):
     requested_for = Column(VARCHAR(length=256), nullable=False)
     requested_at = Column(TIMESTAMP(timezone=True), default=datetime.utcnow, nullable=False)
     completed_at = Column(TIMESTAMP(timezone=True), nullable=True)
+    message = Column(String(length=100), nullable=True)
+    vm_connections = Column(JSON(), nullable=True)
```

Comment:
I would suggest to use `JSONB` just in case we will need to index/query this data later.
```suggestion
    vm_connections = Column(JSONB(), nullable=True)
```

Reply:
Done